File: Python/account.py
class Account:
    id           = int
    name         = str
    document     = str
    email        = str
    password     = str

    #self es como el .this. Hace referencia a los datos que componen la clase
    def __init__(self, name, document):
        self.name      = name
        self.document  = document

File: PHP/account.php
<?php

class Account {
public $id;
public $name;
public $document;
public $email;
public $password;
              #nombre de la funcion
    public function _construct($name, $document){
        $this->name = $name;
        $this->document = $document;
    }

} 

?>
